aiobotocore/awsrequest.py

```diff
@@ -10,7 +11,10 @@ async def _content_prop(self):
 
         if self._content is None:
             # NOTE: this will cache the data in self.raw
-            self._content = await self.raw.read() or b''
+            if isinstance(self.raw, httpx.Response):
+                self._content = await self.raw.aread() or b''
```

Comment:
(side-note) IMO this was a silly design decision of httpx. They should have had two base classes, one sync, one async...but they merged them together for some unknown reason.

Reply:
Is it fine to resolve this with renaming `tests.test_response.AsyncBytesIO.read` to `aread` instead of doing `isinstance`/`hasattr`/`try:except` in AioAWSResponse - or is that an API change in some indirect way?